utils/bonusCode.ts

```diff
@@ -2,9 +2,12 @@ import { randomBytes } from "crypto";
 import { promisify } from "util";
 
 import { isLeft } from "fp-ts/lib/Either";
-import * as t from "io-ts";
 
-import { PatternString } from "italia-ts-commons/lib/strings";
+import { BonusCode } from "../generated/definitions/BonusCode";
```

Comment:
Same here: `definitions` are for api, should we consider domain types here?

```suggestion
import { BonusCode } from "../generated/models/BonusCode";

Reply:
here imho does not matter since the conversions method (+ static types) will take care